### PR TITLE
First draft of rule language formalisation

### PR DESCRIPTION
**Pull request by pdw-xbrl**
This PR provides updates to formalise the English language description of each rule.

### CODE REVIEW

File: docs/DQC_IFRS_0008/DQC_0008.md [hunk 1]
```diff
@@ -1,18 +1,22 @@
 # Reversed Calculation
 Rule ID: DQC_0008   
 
-## Rule function
+## Rule description
 
-This rule evaluates whether a calculation relationship in the company's extension is a reversal of the calculation defined in the base taxonomy used for the filing. The rule checks if the element representing the calculation total and the element representing the component are a reversal of those same elements defined in the base taxonomy with the same calculation weight.  
+This rule evaluates whether a calculation relationship in the _report taxonomy_ is a reversal of the calculation defined in the base taxonomy used for the filing.  Where a concept appears as a contributing item to a total in the _base taxonomy_, the concept representing the total cannot appear as a contributing item to that concept in the _report taxonomy_, unless the calculation weight is reversed.
 
-This rule only compares the defined relationship between two elements joined by a single calculation arc in a company's calculation relationship and checks if the opposite of that relationship exists in the base taxonomy. Both elements in the calculation relationship must have a balance attribute of either debit or credit.
```
Comment (pdw-xbrl) [inline docs/DQC_IFRS_0008/DQC_0008.md:8]: > Both elements in the calculation relationship must have a balance attribute of either debit or credit.

I can't see anything in the implementation that checks this.

File: docs/DQC_IFRS_0008/DQC_0008.md [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
-This rule only compares the defined relationship between two elements joined by a single calculation arc in a company's calculation relationship and checks if the opposite of that relationship exists in the base taxonomy. Both elements in the calculation relationship must have a balance attribute of either debit or credit.  
+Note that this rule only checks for direct relationships between concepts, not all ancestors and descendents.
 
 ## Problem solved by the rule
 
 Users expect that the calculation relationships defined between elements in the base taxonomy can be used to identify the meaning of an element. For example when reporting using the IFRS Taxonomy, Mining Assets is a component of Property Plant and Equipment as Property Plant and Equipment  is comprised of Mining Assets, Land Buildings etc.  
  
 However, if the company changed the calculation relationship so that Property Plant and Equipment was a component of Mining Assets then a user of the data will have a different meaning of the concept of that reported by the preparer.  
 
+## Rule specification
```
Comment (pdw-xbrl) [inline docs/DQC_IFRS_0008/DQC_0008.md:16]: The rule implementation also has:

`and $extCalcSource != $extCalcTarget`

What is this guarding against?

File: docs/DQC_IFRS_0008/DQC_0008.md (hunk 1 above)
Comment (pdw-xbrl) [reply docs/DQC_IFRS_0008/DQC_0008.md:8]: Agreed that this is not part of the rule.

Comment (pdw-xbrl) [reply docs/DQC_IFRS_0008/DQC_0008.md:8]: Agreed that this isn't checked by the rule, so can be removed.

File: docs/DQC_IFRS_0092/DQC_0092.md [hunk 3]
```diff
@@ -1,23 +1,82 @@
...
+
+For each fact with a concept in the list of **non-negative concepts** defined in DQC.IFRS.0080, and a value of EliminationOfIntersegmentAmountsMember for the SegmentConsolidationItemsAxis, the value MUST be less than or equal to zero unless:
+
+* the fact has a dimension with a non-default value that is in the set of **non-positive excluded axis/member pairs**; or
+* the fact has a dimension and member that is in the set of **excluded axes**; or 
+* any string in **non-positive sub-string exclusions** is a case-insensitive substring of the local name of any non-default dimension member applied to the fact.
+
+### Part 2
+
+Let **asset-related concepts** be the list of concepts defined in [dqc_0091_ifrs_2019_concepts.csv](https://github.com/DataQualityCommittee/dqc_us_rules/blob/master/dqc_us_rules/resources/DQC_IFRS_0091/dqc_0091_ifrs_2019_concepts.csv).
```
Comment (pdw-xbrl) [inline docs/DQC_IFRS_0092/DQC_0092.md:66]: This is referencing a 2019 list of concepts.  Does this need to depend on the IFRS taxonomy version?

Should this list be maintained in the DQR repository?

File: docs/DQC_IFRS_0008/DQC_0008.md (hunk 2 above)
Comment (campbellpryde) [reply docs/DQC_IFRS_0008/DQC_0008.md:16]: This is a catch all, which prevents loops.  This is not required in a rule implementation if the instance is XBRL valid.

**Pull request merged**